File: Week1/Pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void print_space(int n);
void print_hash(int n);

int main(void)
{
    int height;
    do{
        printf("Height: ");
        height = get_int();
    }while(height < 0 || height > 23);
    
    for(int i = 0; i < height; i++)
    {
        print_space(height-i-1);
        print_hash(1+i);
        printf("  ");
        print_hash(1+i);
        printf("\n");
    }
    
    
 
}

void print_space(int n)
{
    for(int i = 0; i < n; i++)
    {
        printf(" ");
    }
}

void print_hash(int n)
{
    for(int i = 0; i < n; i++)
    {
        printf("#");
    }
}

File: Week08/Pset6/mario.py
def print_space(n):
    for i in range(n):
        print(" ", end="")

def print_hash(n):
    for i in range(n):
        print("#", end="")

while True:
    try:
        Height = int(input("Height: "))
        if Height > 0 and Height <= 23:
            break
    except ValueError:
        pass

for i in range(Height):
    print_space(Height - 1 - i)
    print_hash(i + 1)
    print_space(2)
    print_hash(i + 1)
    print()